client/modules/IDE/components/ConsoleInput.jsx

```diff
@@ -1,39 +1,33 @@
+import React, { useRef, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import React from 'react';
 import CodeMirror from 'codemirror';
 import { Encode } from 'console-feed';
 
 import RightArrowIcon from '../../../images/right-arrow.svg';
 import { dispatchMessage, MessageTypes } from '../../../utils/dispatcher';
 
-// heavily inspired by
-// https://github.com/codesandbox/codesandbox-client/blob/92a1131f4ded6f7d9c16945dc7c18aa97c8ada27/packages/app/src/app/components/Preview/DevTools/Console/Input/index.tsx
+const ConsoleInput = ({ theme, dispatchConsoleEvent, fontSize }) => {
+  const [commandHistory, setCommandHistory] = useState([]);
+  const [commandCursor, setCommandCursor] = useState(-1);
```

Comment:
Let's put a `// eslint-disable-next-line no-unused-vars` above this line to suppress the warning.  It's definitely not the best code to have a state which isn't used but that's not your fault, that's how it is currently.